application/controllers/QuestionAdministrationController.php

```diff
@@ -2243,12 +2234,12 @@ private function getGeneralOptions(
         $iQuestionId = null,
         $sQuestionType = null,
         $gid = null,
-        $question_template = 'core'
+        $themeName = 'core'
```

Comment:
Shouldn't this be questionThemeName?

Reply:
"core" is not a theme name, it's a theme _type_.

Reply:
> "core" is not a theme name, it's a theme _type_.

This issue that you mention is something that it is carried from prior implementations.
On this change we just focus on porting the question attribute (on a key-value table) to an attribute of the class itself.
The contents of the attribute should remain the same.

We can think of normalizing that later maybe?

Reply:
We will review when "core" is expected for the theme_name attribute.
We believe it is not necessary and that it is legacy code, but want to double check it

Reply:
> The contents of the attribute should remain the same.

If this is possible, then sure. :)